test/component-viewmodel-test.js

```diff
@@ -62,30 +62,30 @@ helpers.makeTests("can-component viewModels", function(){
 
 	QUnit.test("a SimpleMap constructor as viewModel", function() {
 		var MyMap = SimpleMap.extend({
-			setup: function(props){
-                props.name = "Matthew";
-                return SimpleMap.prototype.setup.apply(this, arguments);
-            }
-		});
+		 setup: function(props){
+			props.name = "Matthew";
+			return SimpleMap.prototype.setup.apply(this, arguments);
+		}
+	});
 
 		Component.extend({
-			tag: "can-map-viewmodel",
-			view: stache("{{name}}"),
-			viewModel: MyMap
-		});
+		 tag: "can-map-viewmodel",
```

Comment:
I assume the spacing is already wack.  Is this cleaning it up or making it wackier?  It seems like it's adding spaces.

Reply:
It was pretty screwy, so I ran reindent in Sublime to clean it up a bit.